helm_deploy/prison-offender-events/values.yaml

```diff
@@ -27,7 +27,7 @@ generic-service:
     AWS_REGION: "eu-west-2"
     SPRING_PROFILES_ACTIVE: "postgres,sns"
     APPLICATIONINSIGHTS_CONNECTION_STRING: "InstrumentationKey=$(APPINSIGHTS_INSTRUMENTATIONKEY);IngestionEndpoint=https://northeurope-0.in.applicationinsights.azure.com/;LiveEndpoint=https://northeurope.livediagnostics.monitor.azure.com/"
-    TEST_WIND_BACK_SECONDS: "10"
+    TEST_WIND_BACK_SECONDS: "0"
```

Comment:
not sure I understand why this needs to change for the experiment ?

Reply:
It doesnt need to, its just the "strictest" value so if everything is ok it means using DEQ is bulletproof

Reply:
Actually I got cold feet about the prospect of editing the live deployment to vary the value!